node-tests/blueprints/initializer-test-test.js

```diff
@@ -62,16 +69,99 @@ describe('Blueprint: initializer-test', function() {
 
   describe('in addon', function() {
     beforeEach(function() {
-      return emberNew({ target: 'addon' }).then(() =>
-        generateFakePackageManifest('ember-cli-qunit', '4.1.0')
-      );
+      return emberNew({ target: 'addon' });
+    });
+
+    describe('with ember-cli-qunit@4.1.0', function() {
+      beforeEach(function() {
+        generateFakePackageManifest('ember-cli-qunit', '4.1.0');
+      });
```

Comment:
The [tests in emberjs/ember.js](https://github.com/emberjs/ember.js/blob/31ed8a985dc2c711bc2c807a7f6d50b16b6de22b/node-tests/blueprints/initializer-test-test.js#L70-L82) do not include this. Perhaps it's an improvement, though?

Reply:
Ah yes, I guess I should update the test in ember.js as well. This is better in general, and actually needed here, because without explicitly specifying the used `ember-cli-qunit` version, this addon will default to 4.3.2+ (because that's what in its `package.json`). That's a weird thing, so basically I added the explicit test framework dependency everywhere to make sure the tests do not depend on its own version. See https://github.com/emberjs/ember.js/pull/17126#issuecomment-431071593 where this came up as well...

So * tl;dr*: this should be fine! 😆

Reply:
FYI: https://github.com/emberjs/ember.js/pull/17223

Reply:
Also: https://github.com/emberjs/ember.js/pull/17222

Reply:
Nice, thanks! 👍